liferay-platform-experience/liferay-portal
# LPD-32127 Centralize the logic to import .properties files for a given language id

Ticket: https://liferay.atlassian.net/browse/LPD-32127

## Motivation
Currently we have the same logic to import a .properties file using the language override functionality (UI) or the Language headless API. 

## Proposed Solution
Centralize the logic into the PLOEntryLocalService. We're going to use this for the Language client extensions that will be created in the next tasks see https://liferay.atlassian.net/browse/LPD-30343

## Steps to Verify
Scenario 1: importing a Language.properties file using the headless API

1. Go to localhost:8080/o/api
2. Find the Language REST application
3. find the /messages/import endpoint and submit a file
4. Use the GET method to get a translation

Scenario 2: importing a Language.properties file using the UI

1. Go to control panel > Language Override
2. Click on the three dots on the upper right side
3. Import a Language.properties file
5. Verify the languages were added

## Note
There is no test since I'm not adding any new functionality, just refactoring
